meta/main.yml

```diff
@@ -7,28 +7,18 @@ galaxy_info:
   platforms:
     - name: Ubuntu
       versions:
-        - lucid
-        - maverick
-        - natty
-        - oneiric
-        - precise
-        - quantal
-        - raring
-        - saucy
         - trusty
-        - utopic
-        - vivid
         - xenial
+        - artful
+        - bionic
     - name: Debian
       versions:
-        - squeeze
-        - wheezy
```

Comment:
IMO we should keep `wheezy` support, it'll be EOL'ed soon but is not yet. Unless there's a specific reason you want to remove it?

Reply:
I thought it was alreadt EOL, but indeed `31st May 2018` !
Good catch, thanks !